eclib/JWord.ec

```diff
@@ -874,6 +874,8 @@ abbrev (`&`) = andw.
 abbrev (`|`) = orw.
 abbrev (`^`) = (+^).
 
+op msb (b: t): bool = modulus <= (to_uint b).
```

Comment:
Are you sure its `modulus` and not its half? You may also test if the signed interpretation is negative, or simply access the most significant bit (`b.[size - 1]`).

Reply:
You're right, I messed that up somehow... 
I went with the unsigned comparison, at least in the case of the Kyber proof, this make it easier to reason about the behaviour of the code. 
Maybe it makes sense to add some auxiliary lemmas that state the other properties you mentioned (i.e. if the `msb` is set, then the signed representation is negative or that the result of `msb` is equal to that of indexing the most significant bit directly). I can include those here too